### PR TITLE
Support SD Elements versions prior to 4.0

### PR DESCRIPTION
**Pull request by sibartlett**

### CODE REVIEW

File: sdetools/alm_integration/alm_plugin_base.py [hunk 1]
```diff
@@ -749,6 +749,16 @@ def _determine_latest(self, alm_task, sde_task):
 
         return 'sde' if sde_time > alm_time else 'alm'
 
+    def _check_supports_references(self, tasks):
+        if not tasks:
+            return
+
+        for task in tasks:
+            if 'references' in tasks:
```
Comment (geoffwhittington) [inline sdetools/alm_integration/alm_plugin_base.py]: `if 'references' in task:` I think?

Comment (sibartlett) [inline sdetools/alm_integration/alm_plugin_base.py]: Hah, you're right!

**Comment by geoffwhittington** [comment]
This looks like it should work assuming that `references` is always returned for v4.0 servers in tasks, and I think it does. 

:+1:

**Comment by sibartlett** [comment]
I have checked that `references` is always returned :)

**Pull request merged**